### DISCUSSION TITLE
Trip the Sungazer

### DISCUSSION THREAD

**Discussion by KrabbyCurse** [Techokami/SonicWorldsNext#101] [Ideas]
I would love to see Trip the Sungazer from Sonic Superstars as a playable character for Sonic Worlds Next
[spoilers removed, never do this again -Techo]
Sonic Superstar it's coming tomorrow

**Comment by DimensionWarped** [comment]
Very low priority unless someone drops a complete (as in all standard animations covered and in the correct style) sprite sheet in our laps... in which case it would just be somewhat low priority.

**Comment by Techokami** [comment]
Why the hell did you just post spoilers for a game that's not out yet?
